tox.ini

```diff
@@ -16,11 +18,12 @@ whitelist_externals=
     sh
     rm
     cp
+commands_pre=
```

Comment:
Is this doing anything without a command?

Reply:
It's just overriding the commands_pre from the [testenv] section, setting it back to empty.

[testenv] is defining the default commands_pre for all test environments (including "dynamic" environments which are not listed explicitly, like py37, py38 etc), then these which don't need it are resetting it back to empty.